### PR TITLE
44 Login

### LINKED ISSUE #18: Alert pop-up notifications

### LINKED ISSUE #44: Basic Login to Bitshares
Basic Login to Bitshares

+ prevent unauthorized access
+ validate account name
+ validate fields: both required & show errors

image

### PR DESCRIPTION
**Pull request by roma219**
#### What's this PR do?
Creates a Login Page with cloud/brainkey login & logout functionality & prevents unauthorized access to app. Also introduces a new way of persisted storage for vuex data. Adds pop-up notifications.

closes #44 
closes #18 
#### Where should the reviewer start?
`srv/views/Login.vue`
`src/store/index.js`
#### How should this be manually tested?
Try loggin in cloud password and brainkey & logging out. Refresh page & try private mode for persistent storage testing.
#### Any background context you want to provide?
Make sure to look throught new auth module in `vuex-bitshares` repo - temporarily named `acc`
#### What are the relevant tickets?
#44, https://github.com/bitshares/vuex-bitshares/issues/4
#### Screenshots (if appropriate)
image
image
#### Questions:
@youaresofunny @ololokiras @MindfulMe Guys, please look through code here and in vuex-bitshares to get to know what is going on with project & ask any questions you get 
#### Deployed at
http://staging-ui.trusty.apasia.tech:8080/44-basic-login

### CODE REVIEW

**Comment by youaresofunny** [comment]
image
Hide it mb?

File: src/components/Input/Input.vue [hunk 1]
```diff
@@ -116,7 +131,7 @@ export default {
 
 .input {
   position: relative;
```
Comment (mindfulme) [inline src/components/Input/Input.vue:172]: Can't we use tailwind classes like ".relative"? [source](http://nerdcave.com/tailwind-cheat-sheet)

**Review by mindfulme — APPROVED** [review]

**Review by youaresofunny — DISMISSED** [review]
image
Smth wrong with markup in "Private key login" tab

image
Need "Next" button here (now it's "go" - that's submitting input)

**Comment by roma219** [comment]
@youaresofunny fixed. Android "Next" instead of "Go" - not possible as of now.

File: src/components/Input/Input.vue (hunk 1 above)
Comment (roma219) [reply src/components/Input/Input.vue:172]: We can, but i figured it's actually mostly needed for reusable stuff like fonts/paddings/margins/colors. For other stuff like positioning it's ok both ways.

**Review by ololokiras — COMMENTED** [review]
When I use autocomplete, background change 
image

**Comment by youaresofunny** [comment]
image
Ye, it looks weird with autocomplete

**Review by youaresofunny — COMMENTED** [review]
image
@roma219  got this wrong background behaviour on mobile input open

Also sometimes view jumps to input, sometimes not - but i think we don't need to handle it in this PR.

**Comment by roma219** [comment]
@youaresofunny Fixed number keyboard for mobile, probably fixed autocomplete bg color - if you see it on mobile, please make a separate issue for that.

**Review by youaresofunny — APPROVED** [review]

**Pull request merged**